src/main/java/com/faforever/client/fx/PlatformServiceImpl.java

```diff
@@ -14,8 +16,15 @@
 
   private final HostServices hostServices;
 
+  private boolean isOnWindows;
+
+  @Value("${forgedAlliance.windowTitle}")
+  private String faWindowTitle;
```

Comment:
This value will currently be null, how do I cause it to be initialized, is this even a good idea? I wouldn't pass the faWindowTitle from every code line where I want to focus the game, since this should be (in my opionon) be the task of the PlatformService, this also applies to the focusGameWindow, startFlashingGameWindow, .... below.